tests/Datagrid/ProxyQueryTest.php

```diff
@@ -119,8 +119,8 @@ public function testExecuteAllowsSorting(): void
         $proxyQuery->setSortBy([], ['fieldName' => 'name']);
         $proxyQuery->setSortOrder('DESC');
 
-        /** @var array{array{name: string}} $result */
-        $result = $proxyQuery->execute()->toArray();
+        /** @var iterable<array{name: string}> $result */
```

Comment:
If you typehint the ProxyQuery as `ProxyQuery<array{name: string}>` you won't have to typehint the result.